src/OrderFormAcceptanceTests.TestData/SupplierInfo.cs

```diff
@@ -99,7 +99,7 @@ public static int SupplierWithMoreThanOneSolution(string connectionString)
         public static string GetSupplierSolutionNameWithPrice(string connectionString, string supplierId)
         {
             var query =
-                $@"SELECT ci.[Name] FROM [buyingcatalogue].[dbo].[CatalogueItem] ci INNER JOIN CataloguePrice cp ON cp.CatalogueItemId=ci.CatalogueItemId WHERE ci.CatalogueItemTypeId = 1 AND ci.PublishedStatusId = 3 AND ci.SupplierId = @supplierId";
+                $@" ci.[Name] FROM dbo.CatalogueItem AS ci INNER JOIN CataloguePrice AS cp ON cp.CatalogueItemId = ci.CatalogueItemId WHERE ci.CatalogueItemTypeId = 1 AND ci.PublishedStatusId = 3 AND ci.SupplierId = @supplierId;";
```

Comment:
I'm guessing losing `SELECT` wasn't intentional? Apologies if it was missing from my example suggestion.